lib/backports/2.7.0/time/ceil.rb

```diff
@@ -0,0 +1,10 @@
+unless Time.method_defined?(:ceil)
+  class Time
+    def ceil(ndigits = 0)
+      # Imitate value.ceil(ndigits) which does not exist in Ruby < 2.4
```

Comment:
Oh, interesting... A missing 2.4 backports then... Might as well do it too...

Reply:
I'd prefer not to add it to this PR, float arithmetics is kinda scary (I mean, to do it right AND efficiently) :woman_shrugging: